bin/fqn-check

```diff
@@ -41,20 +41,24 @@ $fileCount = count($files);
 print "Checking {$fileCount} file" . ($fileCount === 1 ? "" : "s") . "…" . PHP_EOL . PHP_EOL;
 
 foreach ($files as $file) {
-    /** @var SplFileInfo $file */
-    $nodes = $parser->parse($file->getContents());
-    $nodeTraverser->traverse($nodes);
+    try {
+        /** @var SplFileInfo $file */
+        $nodes = $parser->parse($file->getContents());
+        $nodeTraverser->traverse($nodes);
 
-    $findings = $fqnChecker->getFindings();
+        $findings = $fqnChecker->getFindings();
 
-    foreach ($findings as $finding) {
-        if ($errors === 0) {
-            print "Found not fully qualified function calls and/or constants in the following files:" . PHP_EOL;
-        }
+        foreach ($findings as $finding) {
+            if ($errors === 0) {
+                print "Found not fully qualified function calls and/or constants in the following files:" . PHP_EOL;
+            }
 
-        $errors++;
+            $errors++;
 
-        print " - " . $file->getRelativePathname() . " on line {$finding["line"]} ({$finding["function"]})" . PHP_EOL;
+            print " - " . $file->getRelativePathname() . " on line {$finding["line"]} ({$finding["function"]})" . PHP_EOL;
+        }
+    } catch (PhpParser\Error $e) {
+        print $e->getMessage() . " (errored in file: " . $file->getFilename() . ")\n";
     }
```

Comment:
If we catch the exception, it needs to increase the error count, otherwise it exits as if everything was fine, no?

Reply:
I don't think so, because just because PhpParser didn't manage to parse the file, doesnt mean there are non-fully-qualified names in the file.

Reply:
But we can do something like: "There was 51 non-FQN files, and 6 files couldn't be parsed".

Reply:
If you want to implement that, sure, go ahead! :-)

Reply:
@kelunik Ok, I'll update this PR. In the meantime, you can take a look at the other one.

Reply:
@kelunik Do you mind if I write unit tests in phpUnit?

Reply:
Sure, go ahead. 😊

Reply:
But there's a problem, the script uses `exit()` to return status code, which terminates the unit tests :/

I would have to extract it to a separate file or a class.

Reply:
@kelunik